README.md

```diff
@@ -341,6 +341,34 @@ monday.execute('openFilesDialog', {
 })
 ```
 
+
+#### Trigger file upload process
+Triggers the file upload flow to a specific file column. Rejects the promise in case of error.
```

Comment:
What does "trigger thhe file upload flow" mean? 

I would suggest something like:
Opens a modal to let the current user upload a file to a specific file column. 

Returns a promise. In case of error, the promise is rejected.